futures-channel/Cargo.toml

```diff
@@ -27,9 +27,5 @@ unstable = ["futures-core/unstable"]
 cfg-target-has-atomic = ["futures-core/cfg-target-has-atomic"]
 
 [dependencies]
-futures-core = { path = "../futures-core", version = "0.3.0", default-features = false }
-futures-sink = { path = "../futures-sink", version = "0.3.0", default-features = false, optional = true }
-
-[dev-dependencies]
-futures = { path = "../futures", version = "0.3.0", default-features = true }
-futures-test = { path = "../futures-test", version = "0.3.0", default-features = true }
```

Comment:
Presumably you don't want to merge the dev-dependency removal?

Reply:
Or are these no longer used?

Reply:
Heh, no, I don't, thanks for the catch.